src/components/Intro.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react-dom'
+import { Line, Dot } from '../styles/Intro'
+import { Section, Wrapper, MainText } from '../styles/Main'
+
+const Intro = () => {
+  return (
+    <Section>
+      <Wrapper>
+        <MainText> <Line /><Dot />I am a creative and driven frontend developer. 
```

Comment:
Maybe add some space between the <dot /> and the start of the text. Right now they're almost bumping into each other.